src/client/dfuse/dfuse_core.c

```diff
@@ -171,6 +171,12 @@ dfuse_start(struct dfuse_info *dfuse_info, struct dfuse_dfs *dfs)
 
 	fs_handle->dpi_info = dfuse_info;
 
+	/* Max read and max write are handled differently because of the way
+	 * the interception library handles reads vs writes
+	 */
+	fs_handle->dpi_max_read = 1024*1024*4;
```

Comment:
Should we define a constant for this, say something with 4MB in the name?

Reply:
Probably.

The kernel actually restricts this to 32 pages or 128k in practice so we could just use that value and comment it appropriately.  I don't think it makes a huge difference today, and given the kernel restrictions I don't see much point in making it configurable either.